<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qxgedit - Qt XG Editor.

   Copyright (C) 2005-2009, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qxgeditMainForm</class>
 <widget class="QMainWindow" name="qxgeditMainForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="acceptDrops" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>QXGEdit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/qxgedit.qrc" >:/icons/qxgedit.png</iconset>
  </property>
  <widget class="QWidget" name="widget" />
  <widget class="QToolBar" name="ToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="fileNewAction" />
   <addaction name="fileOpenAction" />
   <addaction name="fileSaveAction" />
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="fileOpenRecentMenu" >
     <property name="title" >
      <string>Open &amp;Recent</string>
     </property>
    </widget>
    <addaction name="fileNewAction" />
    <addaction name="separator" />
    <addaction name="fileOpenAction" />
    <addaction name="fileOpenRecentMenu" />
    <addaction name="separator" />
    <addaction name="fileSaveAction" />
    <addaction name="fileSaveAsAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="viewMenu" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="viewMenubarAction" />
    <addaction name="viewStatusbarAction" />
    <addaction name="viewToolbarAction" />
	<addaction name="separator" />
    <addaction name="viewOptionsAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction" />
    <addaction name="helpAboutQtAction" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="viewMenu" />
   <addaction name="separator" />
   <addaction name="helpMenu" />
  </widget>
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset resource="../icons/qxgedit.qrc" >:/icons/fileNew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="iconText" >
    <string>New</string>
   </property>
   <property name="toolTip" >
    <string>New session</string>
   </property>
   <property name="statusTip" >
    <string>New session file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset resource="../icons/qxgedit.qrc" >:/icons/fileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="iconText" >
    <string>Open</string>
   </property>
   <property name="toolTip" >
    <string>Open session</string>
   </property>
   <property name="statusTip" >
    <string>Open session from file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAction" >
   <property name="icon" >
    <iconset resource="../icons/qxgedit.qrc" >:/icons/fileSave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="iconText" >
    <string>Save</string>
   </property>
   <property name="toolTip" >
    <string>Save session</string>
   </property>
   <property name="statusTip" >
    <string>Save session to file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save As</string>
   </property>
   <property name="toolTip" >
    <string>Save as</string>
   </property>
   <property name="statusTip" >
    <string>Save current session with another file name</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="toolTip" >
    <string>Exit</string>
   </property>
   <property name="statusTip" >
    <string>Exit this application program</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="viewMenubarAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Menubar</string>
   </property>
   <property name="iconText" >
    <string>Menubar</string>
   </property>
   <property name="toolTip" >
    <string>Menubar</string>
   </property>
   <property name="statusTip" >
    <string>Show/hide the main program window menubar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="viewStatusbarAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Statusbar</string>
   </property>
   <property name="iconText" >
    <string>Statusbar</string>
   </property>
   <property name="toolTip" >
    <string>Statusbar</string>
   </property>
   <property name="statusTip" >
    <string>Show/hide the main program window statusbar</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="viewToolbarAction" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Toolbar</string>
   </property>
   <property name="iconText" >
    <string>Toolbar</string>
   </property>
   <property name="toolTip" >
    <string>Toolbar</string>
   </property>
   <property name="statusTip" >
    <string>Show/hide main program window file toolbar</string>
   </property>
  </action>
  <action name="viewOptionsAction" >
   <property name="text" >
    <string>&amp;Options...</string>
   </property>
   <property name="iconText" >
    <string>Options</string>
   </property>
   <property name="toolTip" >
    <string>Options</string>
   </property>
   <property name="statusTip" >
    <string>Change general application program options</string>
   </property>
   <property name="shortcut" >
    <string>F12</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="toolTip" >
    <string>About</string>
   </property>
   <property name="statusTip" >
    <string>Show information about this application program</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutQtAction" >
   <property name="text" >
    <string>About &amp;Qt...</string>
   </property>
   <property name="iconText" >
    <string>About Qt</string>
   </property>
   <property name="toolTip" >
    <string>About Qt</string>
   </property>
   <property name="statusTip" >
    <string>Show information about the Qt toolkit</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="8" />
 <resources>
  <include location="../icons/qxgedit.qrc" />
 </resources>
 <connections/>
</ui>
